app/src/main/java/mozilla/lockbox/presenter/EditItemPresenter.kt

```diff
@@ -58,11 +58,11 @@ class EditItemPresenter(
     override fun saveChangesAction(hasChanges: Boolean) =
         itemId?.let {
             if (hasChanges) {
-                ItemDetailAction.EditItemSaveChanges
+                listOf<Action>(ItemDetailAction.EditItemSaveChanges)
             } else {
-                null
+                listOf(ItemDetailAction.EndEditItemSession)
```

Comment:
This can be `null`; the combination of the `let` and the `?:` means that the result will be non-null.